samples/CoreAnimationTest/CoreAnimationTest/ProgrammaticViewController.m

```diff
@@ -38,13 +38,13 @@ - (void)viewDidLoad {
     [self addMenuItemViewController:[[CALayerAnimationViewController alloc] init] andTitle:@"CALayer Animation (CAAnimation)"];
 
     // CAShapeLayer
-    [self addMenuItemSegue:@"CAShapeLayer" andTitle:@"CAShapeLayer"];
+    [self addMenuItemSegue:@"CAShapeLayer" andTitle:@"CAShapeLayer - not supported"];
 
     // CAScrollLayer
-    [self addMenuItemSegue:@"CAScrollLayer" andTitle:@"CAScrollLayer"];
+    [self addMenuItemSegue:@"CAScrollLayer" andTitle:@"CAScrollLayer - not supported"];
 
     // CAEmitterLayer
-    [self addMenuItemSegue:@"CAEmitterLayer" andTitle:@"CAEmitterLayer"];
+    [self addMenuItemSegue:@"CAEmitterLayer" andTitle:@"CAEmitterLayer - not supported"];
```

Comment:
Same response as above - we will keep it for now.

---
In reply to: [99437943](https://github.com/Microsoft/WinObjC/pull/1874#discussion_r99437943) [](ancestors = 99437943)